standard/template.lua

```diff
@@ -22,15 +28,22 @@ function Template.safeExpand(frame, title, args, defaultTemplate)
 	end
 end
 
+---@param frame Frame
+---@param title string
+---@param args table
```

Comment:
```suggestion
---@param args table?
```
right?

Reply:
yeah tested it
in definitions the table wasn't optional though so adjusting that too